python/tests/unit/arcticdb/version_store/test_compaction.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import pandas as pd
+
+from arcticdb.version_store import NativeVersionStore
+
+
+@pytest.mark.parametrize("append", (True, False))
+@pytest.mark.parametrize("mode", ("no_prune", "lib_config", "arg"))
+def test_compact_incomplete_prune_previous(mode: str, append: bool, version_store_factory):
```

Comment:
These parametrizations don't cover the lib config setting being true and the arg being false

Reply:
Also, is there an equivalent test for recursive normalizers?

Reply:
> These parametrizations don't cover the lib config setting being true and the arg being false

Will handle that case.

> Also, is there an equivalent test for recursive normalizers?

No. Recursive normalizer was not in scope to begin with and not officially documented anyway.

Reply:
Can you create tickets for those if they don't already exist then?

Reply:
Raised #711